src/main/java/com/buinak/imageapi/controller/ImageController.java

```diff
@@ -29,6 +29,12 @@ public ImageController(ImageService imageService) {
         return ResponseEntity.ok().body(imageService.addImage(name, description, file));
     }
 
+    @GetMapping(path = "getImageLinks")
+    public ResponseEntity<List> getImageLinks(){
+        List<String> imageUrls = imageService.listImageUrls();
```

Comment:
point of this variable? unless logging is used, return .body(imageService.listImageUrls())